src/igvfd/types/sample.py

```diff
@@ -90,7 +90,7 @@ class Sample(Item):
         'lab',
         'sources',
         'sample_terms',
-        'treatments'
```

Comment:
Treatments is a property on the level of biosample and is already specified on the list of biosample audit inheritance so I removed it here.

Reply:
Technical samples inherit audits from sample not biosample (line 680), did we do this on purpose because technical samples don't have treatments/modifications?

Reply:
Yes. also because technical samples are not a biosample subclass, but a sample subclass.